unity/Holo/Assets/Scripts/STLConverter.cs

```diff
@@ -0,0 +1,185 @@
+﻿using Unity;
```

Comment:
Why all classes are in one **cs** file? Please split them into separate files

Reply:
Sorry, done.